src/Telegram.Bot.Signed/Telegram.Bot.Signed.csproj

```diff
@@ -0,0 +1,35 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <TargetFramework>netstandard2.0</TargetFramework>
+    <VersionPrefix>15.0.0</VersionPrefix>
+    <!--<VersionSuffix></VersionSuffix>-->
+    <LangVersion>latest</LangVersion>
+    <GeneratePackageOnBuild>True</GeneratePackageOnBuild>
+    <GenerateDocumentationFile>True</GenerateDocumentationFile>
+    <Title>Telegram Bot API Client</Title>
+    <Description>The Bot API is an HTTP-based interface created for developers keen on building bots for Telegram.</Description>
+    <PackageId>Telegram.Bot.Signed</PackageId>
+    <Authors>RoundRobin,Poulad</Authors>
+    <Copyright>Copyright © Robin Müller 2016</Copyright>
+    <PackageIconUrl>https://raw.githubusercontent.com/TelegramBots/Telegram.Bot/master/package-icon.gif</PackageIconUrl>
+    <PackageProjectUrl>https://github.com/TelegramBots/telegram.bot</PackageProjectUrl>
+    <PackageLicenseUrl>https://raw.githubusercontent.com/TelegramBots/telegram.bot/master/LICENSE.txt</PackageLicenseUrl>
+    <RepositoryUrl>https://github.com/TelegramBots/telegram.bot.git</RepositoryUrl>
+    <PackageTags>Telegram;Bot;Api;Payment;Inline;Games</PackageTags>
+    <SignAssembly>true</SignAssembly>
+    <AssemblyOriginatorKeyFile>key.snk</AssemblyOriginatorKeyFile>
+  </PropertyGroup>
+
+  <!-- when building on AppVeyor -->
+  <PropertyGroup Condition="'$(AppVeyor)' == 'True'">
+    <VersionSuffix Condition="'$(VersionSuffix)' != ''">$(VersionSuffix)-$(APPVEYOR_BUILD_NUMBER)</VersionSuffix>
+  </PropertyGroup>
+  <ItemGroup>
+    <PackageReference Include="Newtonsoft.Json" Version="11.0.2" />
+  </ItemGroup>
+  <ItemGroup>
+    <Compile Include="..\Telegram.Bot\**\*.cs" Exclude="..\Telegram.Bot\obj\**;..\Telegram.Bot\bin\**">
+      <Link>%(RecursiveDir)%(Filename)%(Extension)</Link>
+    </Compile>
+  </ItemGroup>
```

Comment:
can we sign the original assembly itself? i assume there are potential breaking changes.

Reply:
I was mostly following the pattern used by other projects (for example Markdig) to expose the signed version as Markdig.Signed.

I see the [posted guidance](https://docs.microsoft.com/en-gb/dotnet/standard/assembly/strong-named) is now NOT to ship two separate projects, though I think the argument listed there is weak.

As far as I know the only downside is that you shouldn't stop signing the assembly.